apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/CommitResponse.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Google.Cloud.Spanner.Data
+{
+    /// <summary>
+    /// Immutable class representing a commit response.
+    /// </summary>
+    public sealed class CommitResponse
```

Comment:
There is a chance that we'd add additional fields to the response in the future. So in all languages, we're keeping this super generic to make sure we can accommodate any new response fields. Let me know if this might be overkill. I'm just paranoid because in some languages, this became a breaking change.